pkg/helper/byteHelper.go

```diff
@@ -0,0 +1,23 @@
+package helper
```

Comment:
helper is very broad. What about convert ?

this will make your function usage cleaner:

```golang
...
convert.U64ToBytes(123)
...
```

Reply:
All those helper as useless: you are only wrapping one or two lines of code...